src/main/resources/config/application.yml

```diff
@@ -96,6 +96,14 @@ artemis:
                 c_plus_plus:
                     default: "ghcr.io/ls1intum/artemis-cpp-docker:v1.0.0"
 
+        # The following properties are used to configure the Artemis build agent.
+        # The build agent is responsible for executing the buildJob to test student submissions.
+        build-agent:
+            # Name of the build agent. Only lowercase letters, numbers and hyphens are allowed. ([a-z0-9-]+)
+            short-name: "artemis-build-agent-1"
+
+
+
```

Comment:
What happens if I want more than one build agent? Do you need to explicitly give each a name?

Reply:
Yes, then you would need to add this property in the `application.yml` of every build agent